Blaise-Shyaka/My-Mentor
# Ft change user to mentor #170421063

# What does this PR do?

This page allows the admin to change a regular user to a mentor

# Description of tasks to be completed

- Make the page responsive
- The page should have a description of the user
- The page should have a button allowing the admin to change user to mentor

# How should this be manually tested?

- Clone repository, and cd into views
- open admin-user-view.html and check its responsiveness
- Check whether it has a button to change user to mentor and description of the user
